.pre-commit-config.yaml

```diff
@@ -0,0 +1,36 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+
+  - repo: local
+    hooks:
+      - id: clang-format-fix
+        name: clang-format-fix
+        entry: clang-format-18
+        files: ^(src/|include/)
+        types_or: [c++, c]
+        language: system
+        args: ['-i']
+        stages: [manual]
```

Comment:
This is replaced with:
https://github.com/beman-project/exemplar/blob/7c2ee2821bfe0d36a0f60333b104dc4e3a7353bf/.pre-commit-config.yaml#L27